docs/how-to/deploy_charmed_magma_orchestrator.md

```diff
@@ -34,6 +43,11 @@ applications:
 
     This configuration is unsecure because it uses self-signed certificates.
 
+!!! info
+    
+    Elasticsearch is not part of the magma-orc8r bundle and needs to be deployed prior to deploying 
+    the bundle. Elasticsearch needs to support both `http` and `https` requests.
```

Comment:
Here we should point to a document "how-to: integrate charmed orchestrator with Elasticsearch"...and we should create such a document.

Reply:
What if elasticsearch is deployed at a later moment? That should work as well right? Then this overlay wouldn't be of too much help

Reply:
Makes sense. New doc added.